wcfsetup/install/files/acp/update_com.woltlab.wcf_5.3_style.php

```diff
@@ -0,0 +1,132 @@
+<?php
+require('global.php');
+
+use wcf\data\style\StyleEditor;
+use wcf\data\style\StyleList;
+use wcf\util\FileUtil;
+
+/**
+ * @author	Tim Duesterhus
+ * @copyright	2001-2020 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core
+ */
+
+$styleList = new StyleList();
+$styleList->readObjects();
+
+foreach ($styleList as $style) {
+	$style->loadVariables();
+	$variables = $style->getVariables();
+	$styleEditor = new StyleEditor($style);
+	
+	
+	// 1) Move existing asset path folder out of the way.
+	// It's unlikely that one exists, but having an existing folder will create a small mess.
+	if (file_exists($style->getAssetPath())) {
+		rename($style->getAssetPath(), FileUtil::removeTrailingSlash($style->getAssetPath()) . '.old53/');
```

Comment:
The folder is not deleted. Is that what you want?

Reply:
Yes. The update script very intentionally does not delete anything.